packages/block-library/src/navigation-overlay-close/save.js

```diff
@@ -0,0 +1,40 @@
+/**
+ * WordPress dependencies
+ */
+import { useBlockProps, RichText } from '@wordpress/block-editor';
+import { __ } from '@wordpress/i18n';
+
+export default function NavigationOverlayCloseSave( { attributes } ) {
+	const { displayMode, text } = attributes;
+	const showIcon = displayMode === 'icon' || displayMode === 'both';
+	const showText = displayMode === 'text' || displayMode === 'both';
+
+	// eslint-disable-next-line react-compiler/react-compiler
```

Comment:
I've started getting a linter error here about React compiler 😬 Seems to follow current best practices though